Реализованы тесты и сьюты.

MariyaVavilina opened the pull request.

Commit:
Реализованы тесты и сьюты.

khda91 commented on src/test/java/Calculator/Tests/AbstractBaseTest.java:
```diff
@@ -0,0 +1,22 @@
+import org.testng.annotations.BeforeMethod;
+
+public abstract class AbstractBaseTest {
+    protected com.epam.tat.module4.Calculator calculator;
```
А зачем использовать полный импор в данной строкче класса?
`import com.epam.tat.module4.Calculator;`

khda91 commented on src/test/java/Calculator/Tests/CalculatorDivTest.java:
```diff
@@ -0,0 +1,43 @@
+
+public class CalculatorDivTest extends AbstractBaseTest {
+    @Test
+    public void SimpleDivTest(){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorDivTest.java:
```diff
@@ -0,0 +1,43 @@
+    }
+
+    @Test(dataProvider = "divPositiveTestDataProvider")
+    public void PositiveDataProviderDivTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorDivTest.java:
```diff
@@ -0,0 +1,43 @@
+
+
+    @Test(dataProvider = "Negative Div Test Data")
+    public void NegativeDataProviderDivTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorDivTest.java:
```diff
@@ -0,0 +1,43 @@
+    @Test(dataProvider = "divPositiveTestDataProvider")
+    public void PositiveDataProviderDivTest(double a,double b, double expected){
+        double actual = calculator.div(a, b);
+        Assert.assertEquals(actual, expected);
```
Модно добавить `static import` для метода `assertEquals`

khda91 commented on src/test/java/Calculator/Tests/CalculatorMultTest.java:
```diff
@@ -0,0 +1,43 @@
+
+public class CalculatorMultTest extends  AbstractBaseTest {
+    @Test
+    public void SimpleMultTest(){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorMultTest.java:
```diff
@@ -0,0 +1,43 @@
+    }
+
+    @Test(dataProvider = "multPositiveTestDataProvider")
+    public void PositiveDataProviderMultTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorMultTest.java:
```diff
@@ -0,0 +1,43 @@
+
+
+    @Test(dataProvider = "Negative Mult Test Data")
+    public void NegativeDataProviderMultTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorSubTest.java:
```diff
@@ -0,0 +1,43 @@
+
+public class CalculatorSubTest extends AbstractBaseTest {
+    @Test
+    public void SimpleSubTest(){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorSubTest.java:
```diff
@@ -0,0 +1,43 @@
+    }
+
+    @Test(dataProvider = "subPositiveTestDataProvider")
+    public void PositiveDataProviderSubTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

khda91 commented on src/test/java/Calculator/Tests/CalculatorSubTest.java:
```diff
@@ -0,0 +1,43 @@
+
+
+    @Test(dataProvider = "Negative Sub Test Data")
+    public void NegativeDataProviderSubTest(double a,double b, double expected){
```
Java Code Convention для наименования методов

Commit:
1) AbstractBaseTest отредактирован по замечанию
2) все остальные калассы были изменены в части наименований методов, а именно "первый символ должен быть в нижнем регистре"
